### DISCUSSION TITLE
Time Series insights panel

### DISCUSSION THREAD

**Discussion by emrahnazif** [directus/directus#13738] [Archive]
Insights panel looks strong place to get some usage data, however I am not able to understand how to use it. 
My simple goal is to create a Time Series dashboard to see how many new items created in a given date range by group precision x. 

Example: I have created 1000 items in the collection A last month and I would like to see a how many items per day/week has been created. (if ID may work for this, in my case I use UUID, so they are not integers.)

When I try to create time series panel, it asks for a value field which I do not have. Also I see many errors in console with "You don't have permission to access this", I believe it is due to wrong configuration. 

So, any idea how to do this simple panel?

Thanks

**Comment by benhaynes** [comment]
@jaycammarano @rijkvanzanten — can we offer guidance, or state any limitations here? We'll want this UX to be completely intuitive and capable.

**Comment by rijkvanzanten** [comment]
The `value` field can be set to the primary key if you want to aggregate on `count` with all items 👍🏻

**Reply by rijkvanzanten** [reply]
cc @erondpowell

**Reply by emrahnazif** [reply]
> The `value` field can be set to the primary key if you want to aggregate on `count` with all items 👍🏻

Good idea but not working in my case, as I use UUID for primary keys :)
Nothing is selectable in my case.

**Reply by jamescammarano** [reply]
Can confirm, time series does not seem to have the same allowed fields functionality as say the metric panel. It doesn't seem to look at what aggregate type you have selected to determine which fields to allow.

**Reply by azrikahar** [reply]
> as I use UUID for primary keys :)
> Nothing is selectable in my case.

Seems like although #13479 was a fix to allow selecting primary keys, UUID ended up being not selectable because of the existing typeAllowList:

https://github.com/directus/directus/blob/9f10d1d97833ba5e1653666a51fe050860cb86b7/app/src/panels/time-series/index.ts#L200

Submitted a fix via #13799 👍 Thanks for drawing attention to it.

**Reply by renancatan** [reply]
Hello, just came here looking for answers cuz I had the same issue, however I didn't find one.
So I started to look for ways to solve it, and I just found a solution:

inside "value field" go to this "Edit Raw Value"
![image](https://user-images.githubusercontent.com/60487892/177209625-95dc36a9-0807-4b29-8776-ac8d6f0056ff.png)

With this pop-up opened:
put your id field inside it:
![image](https://user-images.githubusercontent.com/60487892/177209709-7decd5d4-7ff6-47e7-949e-cf9292469d95.png)

Done :) 

Obs: Was unable to perform count(distinct), but count works perfectly.

**Reply by azrikahar** [reply]
@renancatan thanks for sharing the solution! Seems like it is indeed no longer selectable in `9.13.0`, but I've confirmed it worked in `9.12.2`. Tracking this issue in #14285.
